.github/workflows/deployment-tests.yml

```diff
@@ -58,9 +58,9 @@ jobs:
       - uses: actions/checkout@v3
       - name: Install dependencies on ${{ matrix.platform }}
         run: |
-          python3 -m pip install --upgrade pip
-          python3 -m pip install .[contrib]
-          python3 -m pip install coverage codecov
+          python3.11 -m pip install --upgrade pip
```

Comment:
Unrelated change to fix the Mac OS deployment test, which by default uses Python 3.12 now. However, some of our dependencies are not yet supporting 3.12 by now.